src/languages/graphql.js

```diff
@@ -0,0 +1,79 @@
+/*
+ Language: graphql
+ Category: scripting, protocols, web
+ Author: John Foster [jf990][], and others
+ Description: Highlight GraphQL queries with highlight.js.
+*/
+/** @type LanguageFn */
+export default function(hljs) {
+  const GQL_NAME = '[_A-Za-z][_0-9A-Za-z]*';
+  const GQL_PUNCTUATION = '[\\!\\(\\)\\:\\=\\[\\]\\{\\|\\}]{1}';
+  return {
+    name: "graphql",
```

Comment:
```suggestion
    name: "GraphQL",
```

This should be a "pretty name"/branded name, etc...